### ISSUE TITLE
Create New File page features

### ISSUE THREAD

**Issue by thecodingwizard** [cpinitiative/ide#97]
- [ ] autogenerate placeholder file name
- [ ] store compileroptions for every lang (protect against users accidentally changing language and overriding compiler options)
- [ ] better error messages
- [ ] usaco problem selection

**Comment by AsynchronousAI** [comment]
By `better error messages` do you mean rather than doing JS `prompt()` have text display?

**Comment by thecodingwizard** [comment]
Yeah! Maybe an inline error message under the input field that's incorrect. I'm not really sure.
